test-requirements.txt

```diff
@@ -7,6 +7,8 @@ pytest-random-order
 nbsphinx
 sphinx_rtd_theme
 mypy==1.5.1
+mock
+types-mock
```

Comment:
Ah; this was _just_ removed.  See [PR #3086](https://github.com/Parsl/parsl/pull/3086)

Reply:
Fixed